BlueprintUICommonControls/Sources/Image.swift

```diff
@@ -15,15 +15,21 @@ public struct Image: Element {
     /// not precisely match the size that the element is assigned.
     public var contentMode: ContentMode
 
+    /// iOS 14 added support for Image Descriptions using VoiceOver. This is not always appropriate.
+    /// Set this to `true` to prevent VoiceOver from discribing the displayed image.
+    public var blockAccessibilityDiscription: Bool
```

Comment:
I almost wonder if this should be the default and you should have to _opt in_ to this – it feels really unexpected to me generally for it to do this magically.

Reply:
(Especially since it'll vary by OS version – it seems like the cases where we want the ML Magic are few and far in between)

Reply:
Ok, so I did a deep dive into this and figured out the following.  My research leads me to believe this is a relatively rare situation and we should not be turning this off by default. 
https://docs.google.com/document/d/1nmIoJbVvnde0jJISiJYb-ZPG0kHXJWroP2O7ShMdK-c/edit?usp=sharing